### PR TITLE
document how to add a new mini game (closes #111)

### LINKED ISSUE #111: Document how to add a new mini game
We should add instructions that explain how to add a new mini game. This will help future maintainers.

These instructions would explain what files should be created, and which HTML components are required on all pages (ē.g. `<core-navbar>`)

### PR DESCRIPTION
**Pull request by k-yle**
This pull request closes #111 by documenting how to create a new mini game. 

It explains how to create the `index.html` file and configure the common components like the navbar. 

It also explains how to create a basic end-to-end test with cypress.

[To test this PR, click here](https://github.com/k-yle/KKDZ/blob/111-doc-new-game/CONTRIBUTING.md#adding-a-new-game)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,6 +53,71 @@ A team member will review the pull request as soon as possible.
...
+
+1. Create a new subfolder for your game within the `src` folder. In this example we will call it `chess`
+2. Create an `index.html` file containing this template code:
+
+   ```html
+   <!DOCTYPE html>
+   <html lang="en">
+     <head>
+       <meta charset="UTF-8" />
+       <title>chess</title>
```
Comment (Destiny-02) [inline CONTRIBUTING.md:69]: nit pick: capital `C`

Comment (Destiny-02) [inline CONTRIBUTING.md:69]: Consider adding `<meta name="description" content="A collection of for-fun mini-games" />` as all other mini-games have a description. I think it's still ok to leave it out if this is supposed to be a super basic guide to getting things set up.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -53,6 +53,71 @@ A team member will review the pull request as soon as possible.
...
+
+       Hello world!
+       <script type="module" src="../components/script/main.js"></script>
+     </body>
+   </html>
+   ```
+
+3. Edit the home page to add a link to your new game, by adding the following code to `src/index.html`:
+   ```html
+   <a href="./chess/index.html">Visit the chess game!</a>
```
Comment (Destiny-02) [inline CONTRIBUTING.md:95]: Consider including a note about updating the tests in `cypress/e2e/homepage.cy.js` as visiting the mini-games from the homepage are tested there. If you do, don't forget to add `id="chess-btn"` here. 

A note about how they will need to add an image, a description for the game and apply the same animations would also be good.

**Review by Destiny-02 — CHANGES_REQUESTED** [review]
Thanks for adding this. This will help make it really easy to new contributors to add more mini-games to our homepage. I've added some suggestions as comments to this PR.

**Comment by k-yle** [comment]
thanks for the helpful feedback! To make the homepage tests pass, the instructions have become a bit more complicated

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -53,6 +53,99 @@ A team member will review the pull request as soon as possible.
...
+1. Create a new subfolder for your game within the `src` folder. In this example we will call it `chess`
+2. Create an `index.html` file containing this template code:
+
+   ```html
+   <!DOCTYPE html>
+   <html lang="en">
+     <head>
+       <meta charset="UTF-8" />
+       <title>Chess</title>
+       <meta name="description" content="A collection of for-fun mini-games" />
```
Comment (Destiny-02) [inline CONTRIBUTING.md:70]: nit pick: maybe something like `"add your game description here"` would make more sense

**Review by Destiny-02 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (k-yle) [reply CONTRIBUTING.md]: good spotting, fixed

**Review by Destiny-02 — APPROVED** [review]
LGTM. I'm sure the new contributors will appreciate the example commit. It's also a great way to check that your own instructions work (I considered doing this myself so you saved me some time 😅).

**Pull request merged**